<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(86, 197, 150);</string>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 244, 210);
</string>
       </property>
       <property name="text">
        <string>Gestion d'employ√©s</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50, 157, 156);</string>
       </property>
       <property name="text">
        <string>Gestion de Produits</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 244, 210);</string>
       </property>
       <property name="text">
        <string>Gestion de collectes</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 244, 210);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>380</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 244, 210);</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>420</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 244, 210);</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_pic">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>151</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 244, 210);</string>
       </property>
       <property name="text">
        <string>Accueil</string>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>761</width>
         <height>571</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTabWidget
{
background-color: rgb(123, 228, 149);

}

QTabWidget:tab-bar
{
alignment:center;
}

QTabBar:tab
{
width:100px;
height:50px;
}

QTabBar:tab:selected
{
    
	background-color: rgb(50, 155, 156);
    color:rgb(0,255,0);
}*/

QTabBar:tab:hover
{
	background-color: rgb(109, 202, 131);
}</string>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>CRUD</string>
        </attribute>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>0</y>
           <width>771</width>
           <height>521</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(86, 197, 150);

QTabWidget::pane {
    border: 1px solid #c3c3c3;
    background-color: #f0f0f0;
}

QTabWidget::tab-bar {
    alignment: center;
}

QTabBar::tab {
    background: #c3c3c3;
    color: #333333;
    padding: 2px 12px;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
}

QTabBar::tab:selected {
    background: #007acc;
    color: #ffffff;
}
</string>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_8">
          <attribute name="title">
           <string>ajouter</string>
          </attribute>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>50</y>
             <width>371</width>
             <height>361</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 170, 127);</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="2">
             <widget class="QLineEdit" name="lineEdit_setId">
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly</set>
              </property>
              <property name="maxLength">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="l_idp_3">
              <property name="text">
               <string>ID Produit</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QRadioButton" name="r_carton">
                <property name="text">
                 <string>cartons</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="r_plastique">
                <property name="text">
                 <string>plastiques</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="r_verre">
                <property name="text">
                 <string>verre</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="2">
             <widget class="QDateEdit" name="dateEdit_DE">
              <property name="font">
               <font>
                <italic>false</italic>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhPreferNumbers</set>
              </property>
              <property name="showGroupSeparator" stdset="0">
               <bool>false</bool>
              </property>
              <property name="currentSection">
               <enum>QDateTimeEdit::DaySection</enum>
              </property>
              <property name="displayFormat">
               <string>dd/MM/yyyy</string>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
              <property name="timeSpec">
               <enum>Qt::LocalTime</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="lineEdit_prix"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="l_prix_3">
              <property name="text">
               <string>Prix</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="l_dateentree_3">
              <property name="text">
               <string>Date d'entree</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="l_quantite_3">
              <property name="text">
               <string>Quantite</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLineEdit" name="lineEdit_Quantite">
              <property name="maxLength">
               <number>4</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="l_typep_3">
              <property name="text">
               <string>Type de produit</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>0</y>
             <width>321</width>
             <height>61</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>24</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ajouter Produit</string>
           </property>
          </widget>
          <widget class="QPushButton" name="save">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>420</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>Enregistrer</string>
           </property>
          </widget>
          <widget class="QPushButton" name="Binit">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>420</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>R√©initialiser</string>
           </property>
          </widget>
          <widget class="QPushButton" name="b_modif">
           <property name="geometry">
            <rect>
             <x>450</x>
             <y>420</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
          <widget class="QComboBox" name="select_ID">
           <property name="geometry">
            <rect>
             <x>600</x>
             <y>50</y>
             <width>73</width>
             <height>22</height>
            </rect>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
          <zorder>label_2</zorder>
          <zorder>groupBox_2</zorder>
          <zorder>save</zorder>
          <zorder>Binit</zorder>
          <zorder>b_modif</zorder>
          <zorder>select_ID</zorder>
         </widget>
         <widget class="QWidget" name="tab_7">
          <attribute name="title">
           <string>affichage</string>
          </attribute>
          <widget class="QTableView" name="table_matiereR">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>751</width>
             <height>341</height>
            </rect>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 170, 127);</string>
           </property>
          </widget>
          <widget class="QPushButton" name="b_supp_p">
           <property name="geometry">
            <rect>
             <x>330</x>
             <y>430</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>390</y>
             <width>241</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Identifiant:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_suppID"/>
            </item>
           </layout>
          </widget>
          <widget class="QPushButton" name="Btri">
           <property name="geometry">
            <rect>
             <x>640</x>
             <y>390</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="accessibleName">
            <string/>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>tri</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_recherche">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>201</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="placeholderText">
            <string>Chercher</string>
           </property>
          </widget>
          <widget class="QPushButton" name="b_convPDF">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>390</y>
             <width>101</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>convertir to pdf</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Statistique</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Page</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Page</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>Page</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QPushButton" name="b_update">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>480</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>update</string>
       </property>
      </widget>
      <zorder>pushButton_4</zorder>
      <zorder>pushButton_5</zorder>
      <zorder>pushButton_6</zorder>
      <zorder>label_pic</zorder>
      <zorder>tabWidget</zorder>
      <zorder>pushButton_7</zorder>
      <zorder>pushButton</zorder>
      <zorder>pushButton_3</zorder>
      <zorder>pushButton_2</zorder>
      <zorder>b_update</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
